src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +42,43 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+
+        int dateFromP = convertDate(dateFrom);
+        int dateToP = convertDate(dateTo);
```

Comment:
try to use LocalDate parser instead of creating convert method

Reply:
fixed to LocalDate